packages/server/modules/auth/rest/index.ts

```diff
@@ -83,11 +85,36 @@ export default function (app: Express) {
       }
     }
   })
+  openApiDocument.registerOperation('/auth/accesscode', HttpMethod.GET, {
```

Comment:
Skeptical about all this, seems like a lot of extra verbosity, maintenance burden and some extra duplication for negligible benefit.